### PR TITLE
2017-06-06 changes for Fall Summit 2017

### PR DESCRIPTION
**Pull request by NewMexicoKid**
Moved the already-held O'Reilly InnerSource patterns webinar out of upcoming events while leaving the  link for the replay (but pushing it down to second place in the navigation sidebar). 
Added a bunch of information for the Fall Summit 2017, including a new page for the Call For Presentations that embeds a google docs form.
Added a link for the InnerSource Patterns video.
Added some photos of the rooms and of Naperville.

### CODE REVIEW

**Comment by DuaneOBrien** [comment]
Build looks to be failing. I'll see if I can comment into the files about the needed changes to make it pass.

File: events/isc-fall-2017.md [hunk 1]
```diff
@@ -5,27 +5,50 @@ title: 'InnerSource Commons Summit'
...
+* InnerSource Commons governance
+
+Presenters of accepted papers will be notified July 31st. Note that all presentations given at the Fall Summit, unless otherwise designated by the presenter, will be covered by [the Chatham House Rule](https://www.chathamhouse.org/about/chatham-house-rule).
 
 ### Venue
 
-This Summit is hosted by Nokia at their facility. More info TBA
+This Summit is hosted by [Nokia](http://www.nokia.com/en_int) at their facility at [1960 W. Lucent Lane, Naperville, IL 60563-1594](https://goo.gl/maps/cYHJb2jLb1m). The venue spaces there reserved for the Fall Summit include a flexible, large auditorium, four break-out rooms and an on-site cafeteria. 
+
+<img width="600" src="/InnerSourceCommons/assets/img/nokia_ihn_main.jpg"/>
```
Comment (DuaneOBrien) [inline events/isc-fall-2017.md]: During build time, HTML-Proofer is complaining that this image does not have an alt attribute.

File: events/isc-fall-2017.md [hunk 2]
```diff
@@ -5,27 +5,50 @@ title: 'InnerSource Commons Summit'
...
+
+<img width="600" src="/InnerSourceCommons/assets/img/ihn_auditorium.jpg"/>
```
Comment (DuaneOBrien) [inline events/isc-fall-2017.md]: During build time, HTML-Proofer is complaining that this image does not have an alt attribute.

File: events/isc-fall-2017.md [hunk 3]
```diff
@@ -39,13 +62,30 @@ TBA
 
 ### Area Highlights
 
-TBA
+The [city of Naperville](http://www.naperville.il.us/) is a suburb of Chicago, located about forty minutes west of the Chicago downtown and about forty minutes away from the O'Hare International Airport. It was ranked among the nation's safest cities by USAToday and Business Insider. Naperville was voted the second-best place to live in the United States by Money magazine in 2006. It was rated 1st on the list of best cities for early retirement in 2013 by Kiplinger. As of the 2010 census, the city had a population of 141,853, which was estimated to have increased to 147,100 by July 2015. It is the fifth-largest city in Illinois.
+
+Downtown Naperville features [its own riverwalk](http://www.naperville.il.us/enjoy-naperville/naperville-riverwalk/), [created in 1981 to celebrate the city's Sesquicentennial](http://www.napervilleparks.org/the-riverwalk) (150th anniversary). The park features covered bridges, relaxing fountains and distinctive shepherd's crook lights. 
+
+
+<img width="600" src="/InnerSourceCommons/assets/img/naperville_riverwalk_4755340966_fc7390571a_z_brian_jelonek.jpg"/>
```
Comment (DuaneOBrien) [inline events/isc-fall-2017.md]: During build time, HTML-Proofer is complaining that this image does not have an alt attribute.

File: events/isc-fall-2017.md [hunk 4]
```diff
@@ -39,13 +62,30 @@ TBA
...
+<p style="font-size: 8pt; font-style: italic;">Photo by <a href="https://www.flickr.com/photos/jelonekphotography/4755340966/">Brian Jelonek</a></p>
+<img width="600" src="/InnerSourceCommons/assets/img/naperville_riverwalk_4754699889_719ac628ba_z_brian_jelonek.jpg"/>
```
Comment (DuaneOBrien) [inline events/isc-fall-2017.md]: During build time, HTML-Proofer is complaining that this image does not have an alt attribute.

File: events/isc-fall-2017.md [hunk 5]
```diff
@@ -39,13 +62,30 @@ TBA
...
+<p style="font-size: 8pt; font-style: italic;">Photo by <a href="https://www.flickr.com/photos/jelonekphotography/4754699889/">Brian Jelonek</a></p>
+<img width="600" src="/InnerSourceCommons/assets/img/naperville_riverwalk_4754409033_9389f267ca_z_brian_jelonek.jpg"/>
```
Comment (DuaneOBrien) [inline events/isc-fall-2017.md]: During build time, HTML-Proofer is complaining that this image does not have an alt attribute.

**Comment by NewMexicoKid** [comment]
Should now be good to go.

**Pull request merged**